pywhat/identifier.py

```diff
@@ -1,26 +1,46 @@
 import os.path
-from typing import List, Optional
+from typing import Callable, Optional
 
 from pywhat.distribution import Distribution
+from pywhat.helper import keys
 from pywhat.magic_numbers import FileSignatures
 from pywhat.nameThatHash import Nth
 from pywhat.regex_identifier import RegexIdentifier
 
 
 class Identifier:
-    def __init__(self, distribution: Optional[Distribution] = None):
-        if distribution is None:
+    def __init__(
+        self,
+        *,

```

Comment:
What's `*` here?

Reply:
Arguments following it are forced to be keyword-only.` identify()` already accepts a lot of arguments that do not have any priority, and their order may change in the future, so, in my opinion, it is better to force a user not to use positional arguments. Identifier's` __init__` accepts quite a lot of arguments, too.

Reply:
Fair enough! :)